app/views/events/index.html.erb

```diff
@@ -30,18 +30,26 @@
     </section>
 
     <% if @events_by_type.any? %>
-      <% if @group_presenter.get_into_teaching_events.any? %>
-        <%= render partial: "event_category", locals: { organised_by: t("event_groups.get_into_teaching"), category_groups: @group_presenter.get_into_teaching_events } %>
+      <% if @group_presenter.get_into_teaching_events.values.flatten.any? || @display_empty_types %>
```

Comment:
Perhaps this could be a method on the presenter?

Reply:
Good suggestion 👍 I'll do that now